Management_System/BLL/Services/Interfaсes/IBoardService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DAL.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BLL.Services.Interfaсes;
+
+public interface IBoardService
+{
+    Task<BoardEntity> GetById(Guid id, CancellationToken cancellationToken);
+    Task<IEnumerable<BoardEntity>> GetAll(CancellationToken cancellationToken);
+    Task<BoardEntity> Add(BoardEntity entity, CancellationToken cancellationToken);
+    Task<BoardEntity> Update(Guid id, BoardEntity entity, CancellationToken cancellationToken);
+    Task Delete(Guid id, CancellationToken cancellationToken);
+}
```

Comment:
Perhaps it is better to move this to a generic service.
You can do this later if you want to

Reply:
I'll do it later